### ISSUE TITLE
VO-specific tests for DUNE

### ISSUE THREAD

**Issue by jamesp-epcc** [rucio/rucio#6383]
### Description

We would like to add experiment-specific CI tests for DUNE, similar to the ones that already exist for ATLAS and BelleII. DUNE is in some ways more challenging; while ATLAS and BelleII both use generic container images, DUNE tests will require a MetaCat server and the DUNE policy package, so we will need a custom container image.

To keep things similar to the other tests, it will probably be easiest to have everything in a single container image, which should be feasible, but will require some work to create this image. The alternative would be to use a `docker-compose` type solution involving multiple containers, however this would be a departure from how the other tests work and would introduce complexity to the test workflow.

**Comment by bari12** [comment]
@jamesp-epcc This is beyond the 2 year horizon, I would propose we close this. Any objections?

**Comment by jamesp-epcc** [comment]
@bari12 yes, this should definitely be closed. I've actually implemented this now in the DUNE policy package repo since it's more appropriate to have DUNE-specific code in there. I'd forgotten there was still an open issue for it in Rucio itself.
